### PR TITLE
[TEST] Remove SessionManagerSuite to reduce duplicated tests

### PR DESCRIPTION
**Pull request by yaooqinn**
### _Why are the changes needed?_

SessionManagerSuite re-runs all tests in the super class, it's unnecessary.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by yaooqinn** [comment]
cc @yanghua

**Review by ulysses-you — APPROVED** [review]

**Comment by ulysses-you** [comment]
thanks, merging to master

File: kyuubi-common/src/test/scala/org/apache/kyuubi/operation/TClientTestUtils.scala [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    transport.open()
+    try {
+      f(client)
+    } finally {
+      socket.close()
+    }
+  }
+
+  /**
+   * s shall be [[TFrontendService]]
```
Comment (pan3793) [inline kyuubi-common/src/test/scala/org/apache/kyuubi/operation/TClientTestUtils.scala:48]: Then why declare it as a `FrontendService`?

Comment (yaooqinn) [reply kyuubi-common/src/test/scala/org/apache/kyuubi/operation/TClientTestUtils.scala:48]: unnecessary cast at callerside

**Pull request closed without merging**